packages/wonder-blocks-icon/src/index.ts

```diff
@@ -2,6 +2,7 @@ import Icon from "./components/icon";
 import type {IconAsset, IconSize} from "./util/icon-assets";
 
 export * as icons from "./util/icon-assets";
-
+export {default as PhosphorIcon} from "./components/phosphor-icon";
```

Comment:
**nit:** If this was a named export, you can just do:
```suggestion
export {PhosphorIcon} from "./components/phosphor-icon";
```

Reply:
you already sold me in the previous comment! thanks for the suggestion.